src/AzureDevOpsPolicyConfigurator/Arguments/Commands/CommandBase.cs

```diff
@@ -71,25 +45,8 @@ private void ValidateSettings(TSettings settings)
             {
                 throw new ArgumentValidationException("Argument \"in\" must be provided!");
```

Comment:
Argument exception message seems not correct.

`in` should be `settings.CollectionUrl` here?

Reply:
👍